packages/core/src/components/table/table-header-cell/readme.md

```diff
@@ -20,7 +20,7 @@
 
 | Event     | Description                                                                                                                                                          | Type                                                                                      |
 | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- |
-| `tdsSort` | Sends unique Table identifier, column key and sorting direction to the tds-table-body component, can also be listened to in order to implement custom-sorting logic. | `CustomEvent<{ tableId: string; columnKey: string; sortingDirection: "desc" \| "asc"; }>` |
+| `tdsSort` | Sends unique Table identifier, column key and sorting direction to the tds-table-body component, can also be listened to in order to implement custom-sorting logic. | `CustomEvent<{ tableId: string; columnKey: string; sortingDirection: "asc" \| "desc"; }>` |
```

Comment:
Rouge little change here right? 🥷

Reply:
[Never-ending story...](https://youtu.be/2WN0T-Ee3q4?si=5U6n63KO4fGoo7bq)

Reply:
<a href="https://imgflip.com/i/83roky"><img src="https://i.imgflip.com/83roky.jpg" title="made at imgflip.com"/></a><div><a href="https://imgflip.com/memegenerator">from Imgflip Meme Generator</a></div>

Did I do this "meme"-thing right?